lib/app/features/chat/providers/group_messages_provider.dart

```diff
@@ -0,0 +1,96 @@
+// SPDX-License-Identifier: ice License 1.0
+
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+import 'package:ion/app/features/chat/model/message_author.dart';
+import 'package:ion/app/features/chat/model/message_list_item.dart';
+import 'package:ion/app/features/chat/model/money_message_type.dart';
+import 'package:riverpod_annotation/riverpod_annotation.dart';
+
+part 'group_messages_provider.g.dart';
+
+/// Mocked provider that simulates receiving and mapping data messages
+/// from the server to a model, which describes to UI what should be displayed.
+@riverpod
+Future<List<MessageListItem>> groupMessages(Ref ref) async {
```

Comment:
looks like the `Future` doesn't need in this provider

Reply:
Yep, you are right. Since it mocked, we can remove Future. Done.